languages/wikipron/src/whitelist.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+import sys
+import re
+
+
+def whitelist_reader(path):
+    """Reads in whitelist file"""
+    with open(path, "r") as source:
+        for line in source:
+            yield line.rstrip().split()[0]
+
+
+def filter(tsv_path, phonemes, output_path):
```

Comment:
The function `filter` suggests that it's going to return something but instead it writes out the filtered file too! I would call it something like `filter_and_write` or `filter_write`.

Can you type this function? `tsv_path` and `output_path` are `str`, `phonemes` (should be renamed `phones`, since this should also work on "phonetic" data and that's a generic term that encompasses both) should have type `FrozenSet[str]` (have to import it). And the return type is `None`. Then you can check the typing using `mypy` (install it from `pip` then run `mypy whitelist.py`.)